lib/fixtures/keycloak.go

```diff
@@ -845,7 +845,7 @@ func createTokenExchange(ctx context.Context, connectParams *KeycloakConnectPara
 	}
 	if err := client.UpdatePermissionScope(ctx, token.AccessToken, connectParams.Realm,
 		*realmManagementClientID, tokenExchangePolicyScopePermissionID, permissionScopePolicyRepresentation); err != nil {
-		slog.Error("Error creating permission scope : %s", err)
+		slog.Error("Error creating permission scope", "error", err)
```

Comment:
@dmihalcik-virtru Can we stick with key/value. 

```suggestion
		slog.Error("Error creating permission scope", slog.Any("error", err))
```
or `slog.String("error",err.Error())

We can use this linter to enforce the style we want going forward. https://github.com/go-simpler/sloglint

Reply:
Why?